build.cmd

```diff
@@ -478,12 +478,11 @@ if '%BUILD_PROTO_WITH_CORECLR_LKG%' == '0' (
 rem Build Proto
 if '%BUILD_PROTO%' == '1' (
   if '%BUILD_PROTO_WITH_CORECLR_LKG%' == '1' (
-
-    pushd .\lkg & %_dotnetexe% restore &popd
-    @if ERRORLEVEL 1 echo Error: dotnet restore failed  && goto :failure
-
-    pushd .\lkg & %_dotnetexe% publish project.json -o %~dp0\Tools\lkg -r win7-x64 &popd
-    @if ERRORLEVEL 1 echo Error: dotnet publish failed  && goto :failure
+echo on
+    pushd .\lkg\fsc & %_dotnetexe% restore & popd & if ERRORLEVEL 1 echo Error:%errorlevel% dotnet restore failed & goto :failure
```

Comment:
This error will never trigger because `ERRORLEVEL` will have been set to `0` by `popd` which should always succeed.  Each of these lines should be split out into:

``` batch
pushd .\lkg\fsc
%_dotnetexe% restore
if ERRORLEVEL 1 (
    echo Error:%errorlevel% dotnet restore failed
    goto failure
)
popd
```